docs/content/preview/yugabyte-cloud/cloud-clusters/managed-read-replica.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Read replicas
+linkTitle: Read replicas
+description: Scale YugabyteDB Managed clusters.
+headcontent: Reduce read latencies in remote regions
+beta: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+menu:
+  preview_yugabyte-cloud:
+    identifier: managed-readreplica
+    parent: cloud-clusters
+    weight: 150
+type: docs
+---
+
+If your user base is geographically distributed, you can add [read replicas](../../cloud-basics/create-clusters-topology/#read-replicas) to improve read latency in regions that are far from your primary region.
+
+Read Replicas are a read-only extension to the primary cluster. With read replicas, the primary data of the cluster is copied across one or more nodes in a different region. Read replicas do not add to write latencies because writes aren't synchronously replicated to replicas - the data is replicated to read replicas asynchronously.
+
+Each read replica cluster can have its own replication factor. The replication factor determines how many copies of your primary data the replica has; multiple copies ensure the availability of the replica in case of a node outage. Replicas do not participate in the primary cluster RAFT consensus, and do not affect the fault tolerance of the primary cluster or contribute to failover.
+
+You can delete, modify, and scale read replicas. Adding or removing nodes incurs a load on the replica. Perform scaling operations when the replica isn't experiencing heavy traffic. Scaling during times of heavy traffic can temporarily degrade performance and increase the length of time of the scaling operation.
+
+The **Regions** section on the cluster **Settings** tab summarizes the cluster configuration, including the number of nodes, vCPUs, memory, and disk per node, and VPC for each region of the primary cluster and its replicas.
+
```

Comment:
Maybe worth mentioning what we are trying to do here before getting to Prerequisites. I initially didn’t understand what they were for till I scrolled to the Add or edit Read replica.